packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -22,16 +23,67 @@ export default class CardAdoptionChain extends Component<Signature> {
       .card-adoption-chain {
         background-color: var(--boxel-200);
       }
+      .content-with-line {
+        position: relative;
+        transform: translateX(calc(var(--boxel-sp-sm) * -1));
+
+        display: flex;
+        align-items: center;
+        width: calc(100% + calc(var(--boxel-sp-sm) * 2));
```

Comment:
The `transform` and `width` here don't seem to be doing anything

Reply:
It is doing something. Without the `width`, the line won't span the entire schema editor panel; it will be affected by the `padding` of the schema editor panel. Likewise, without the `transform`, the line won't extend to the left side of the schema editor panel.

Screenshot:
- Without the `transform` and `width`.

![Screenshot 2023-10-05 at 15 23 06](https://github.com/cardstack/boxel/assets/12637010/abaed45d-d1c4-422e-8e14-30d1c79d7f4b)

- Without the `transform`.

![Screenshot 2023-10-05 at 15 25 15](https://github.com/cardstack/boxel/assets/12637010/2c707886-4a69-44ee-af53-3146c0301b63)